tests/pallas/pallas_test.py

```diff
@@ -1726,7 +1726,7 @@ def f_ref(q, k, v):
     dq_ref, dk_ref, dv_ref = jax.grad(f_ref, argnums=(0, 1, 2))(q, k, v)
     # TODO(sharadmv): Fix test.
     np.testing.assert_allclose(dq, dq_ref, atol=0.14)
-    np.testing.assert_allclose(dk, dk_ref, atol=0.13)
+    np.testing.assert_allclose(dk, dk_ref, atol=0.1328)
```

Comment:
Thanks! I think four significant figures is a bit much for a tolerance... how about `0.14`?

Reply:
Sure, will update. I actually started out with `0.14` but then thought I'd go as tight as possible :)

Reply:
Done!